configs/sx8m_config

```diff
@@ -1808,7 +1808,7 @@ CONFIG_TOUCHSCREEN_PROPERTIES=y
 # CONFIG_TOUCHSCREEN_INEXIO is not set
 # CONFIG_TOUCHSCREEN_MK712 is not set
 # CONFIG_TOUCHSCREEN_PENMOUNT is not set
-CONFIG_TOUCHSCREEN_EDT_FT5X06=y
+CONFIG_TOUCHSCREEN_EDT_FT5X06=m
```

Comment:
Raymond made most of the these changes, so I'm not quite sure what it related to. It could be that these were changes still on the CES-719 branch when he branched off